astropy/modeling/utils.py

```diff
@@ -484,3 +484,59 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image):
```

Comment:
This function doesn't really belong in here. modeling.utils is mostly for utilities for implementation details of the modeling package itself, and not functions that users would use.
Instead, what we will ultimately want to do (as I've referenced elsewhere) is bake in evaluation of models using their boundaries into Model.__call__.